### ISSUE TITLE
Hover states and anchor tags defined in .svg file don't work in the rendered readme

### ISSUE THREAD

**Issue by BingeCode** [nikolalsvk/nikolalsvk#2]
Since Github only renders an `img` tag and automatically adds an `a` tag to the `img` (unless you wrap the `img` yourself with an anchor). 

I wonder: Did this ever work? I wish there was a way to individually hover over elements in the svg and trigger some animation. That would make for some incredible READMEs. As for now, the only thing we can do is have the animations run instantly and on repeat.

**Comment by nikolalsvk** [comment]
Yes, that's true, you can't click links in the GitHub README that renders an SVG. You can click links in the SVG itself e.g. here https://raw.githubusercontent.com/nikolalsvk/nikolalsvk/main/welcome.svg
